script.js

```diff
@@ -0,0 +1,59 @@
+const books = [
+  {
+    Id: 1,
+    Title: 'Lorem ipsum',
+    Author: 'Testeroo Testyy',
+  },
+  {
+    Id: 2,
+    Title: 'Second Books',
+    Author: 'Testeroo Testyy',
+  },
+];
+
+// Update books from local storage
+if (localStorage.getItem('myBooklist') != null) {
+  const myBooklist = JSON.parse(localStorage.getItem('myBooklist'));
+  books.length = 0;
+  books.push(...myBooklist);
+}
+
+// Display books
+function displayBooks() {
+  let book = '';
+  for (let i = 0; i < books.length; i += 1) {
+    book += `<p>${books[i].Title}</p>`;
+    book += `<p>${books[i].Author}</p>`;
+    book += `<button onclick="removeBook(${books[i].Id})">Remove</button><br>`;
+    book += '<hr>';
+  }
+  document.getElementById('display').innerHTML = book;
+}
+displayBooks();
+
+// Add books
+function addBook() {
+  const title = document.getElementById('Title').value;
+  const author = document.getElementById('Author').value;
+  if (title && author) {
+    const bookId = books.length + 1;
+    books.push({ Id: bookId, Title: title, Author: author });
+    displayBooks();
+    document.querySelector('form').reset();
+    localStorage.setItem('myBooklist', JSON.stringify(books));
+  }
+}
+addBook();
+
+// Remove Book
+function removeBook(removeId) {
+  const filter = books.filter((remBook, i) => {
+    if (removeId === remBook.Id) {
+      books.splice(i, 1);
+      displayBooks();
+    }
+    return true;
+  });
+  return filter;
+}
```

Comment:
- [Optional] Great Job for updating to add data to local storage, I think you forget to delete also from local storage when data is removed which leads getting deleted data on refresh, so update the local storage on each remove operation to fix this issue 👍